sdk/src/vote_program.rs

```diff
@@ -66,12 +66,12 @@ impl Lockout {
 
 #[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
 pub enum VoteInstruction {
-    /// Register a new "vote account" to represent a particular validator in the Vote Contract,
-    /// and initialize the VoteState for this "vote account"
-    /// * Transaction::keys[0] - the validator id
-    /// * Transaction::keys[1] - the new "vote account" to be associated with the validator
-    /// identified by keys[0] for voting
-    RegisterAccount,
+    /// Initialize the VoteState for this "vote account"
+    /// * Transaction::keys[0] - the staker id that is also assumed to be the delegate by default
```

Comment:
This comment needs some work. While the *transaction* containing this instruction may have two accounts, this *instruction* should only point to the second one (the vote account).  The transaction would be a `SystemInstruction::CreateAccount` and `VoteInstruction::InitializeAccount`.

Reply:
@garious, I will need both accounts being passed in to the VoteProgram. To assign the staker as the delegate.

Reply:
Fairly certain you don't. Should be the same pattern as here: https://github.com/solana-labs/solana/blob/master/programs/native/rewards_api/src/rewards_transaction.rs#L49

Reply:
@garious, maybe I'm missing something. I need to assign the "staker" as the delegate by default. I cannot do that if I don't have the staker's pubkey (provided by accounts[0] right now). I suppose I could take it as an argument to Initialize?